disconnected/installing.adoc

```diff
@@ -23,7 +23,9 @@ ifndef::openshift-origin[]
 [id="installing-agent_{context}"]
 == Installing a cluster with the Agent-based Installer
 
-To learn more about installing a cluster in a disconnected environment with the Agent-based installer, see the following procedure:
+To learn more about installing a cluster in a disconnected environment with the Agent-based installer, see the following pages:
+
+* xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc#understanding-disconnected-installation-mirroring[Understanding disconnected installation mirroring]
 
 * xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-with-agent-based-installer[Installing an {product-title} cluster with the Agent-based Installer]
```

Comment:
Can the link to the procedural page be kept here since users will still need to follow it to eventually try installing a cluster? Or should it also be removed?